service-admin/src/App/templates/app/user-search.phtml

```diff
@@ -54,63 +54,67 @@ $this->addErrorMap([
             <h2 class="heading-medium">User details</h2>
 
             <ul>
-                <li>
-                    <strong class="bold-small">Account status: </strong>
-
-                    <?php if (array_key_exists('isActive', $user)) { ?>
-                        <span><?= ($user['isActive'] ? 'Activated' : 'Not activated'); ?></span>
-                    <?php } elseif (array_key_exists('isDeleted', $user) && $user['isDeleted']) { ?>
-                        <span>Deleted</span>
-                    <?php } ?>
-                </li>
-
-                <?php if (isset($user['activatedAt'])) { ?>
+                <div class="user-search-result">
```

Comment:
A `<div>` shouldn't be nested inside a `<ul>`. Is there a reason this markup needs to change?

Reply:
I didn't know that was a rule, I originally did it for the cypress tests and to conform to user-find, but I then should have had the `<ul>` nested inside the `<div>` in that case. I'll change it to that unless there's preference for having the class on the `<ul>`?